algorithm/cgal/section.test.js

```diff
@@ -135,8 +135,8 @@ test('Coplanar section of polygon is the polygon.', (t) => {
     plane: [0, 0, 1, 0],
   };
   const sections = section(
-    [geometry],
-    [{ matrix: blessed([...identityMatrix]) }]
+    [geometry, { type: 'points', matrix: blessed([...identityMatrix]) }],
```

Comment:
What does it mean for a matrix to be blessed?

Reply:
We throw errors on unblessed matrices to try to track down where they're getting produced.

At this point it can probably be retired.